Modix.Bot/Modules/MessageContentPatternModule.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System.Threading.Tasks;
+using Discord.Commands;
+using Modix.Bot.Extensions;
+using Modix.Data.Models.Core;
+using Modix.Services.Blocklist;
+
+namespace Modix.Bot.Modules
+{
+    [Name("Patterns")]
+    [Summary("Use and maintain message content patterns.")]
+    [Group("pattern")]
+    [Alias("patterns")]
+    public class MessageContentPatternModule : ModuleBase
+    {
+        private readonly IMessageContentPatternService _messageContentPatternService;
+
+        public MessageContentPatternModule(IMessageContentPatternService messageContentPatternService)
+        {
+            _messageContentPatternService = messageContentPatternService;
+        }
+
+        [Command("block")]
+        [Summary("Adds a new pattern to block.")]
+        public async Task BlockAsync([Summary("Regex pattern for the blocked content."), Remainder] string pattern)
+        {
+            if (await _messageContentPatternService.DoesPatternExist(Context.Guild.Id, pattern))
```

Comment:
Somewhat minor, but `AddPattern` is already doing this check internally, so we're querying the DB more often than we have to for this command. Would it make sense to just let the service do this check?

Reply:
Yes, absolutely. Had this in here before I added `ServiceResponse`